Roles/Impostor/Exorcist.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Intrinsics.Arm;
+using System.Text;
+using System.Threading.Tasks;
+using TOHE.Roles.Core;
+using UnityEngine;
+using static TOHE.Translator;
+
+namespace TOHE.Roles.Impostor;
+
+internal class Exorcist : RoleBase
+{
+    //===========================SETUP================================\\
+    private const int Id = 30200;
```

Comment:
30800 so i dont have to offset all my coven ids
(I edited this in the liveshare I don't know why it didn't change)